src/Sets/Zonotope.jl

```diff
@@ -406,3 +406,24 @@ function _split(Z::Zonotope, gens::AbstractVector, n::AbstractVector)
     end
     return Zs
 end
+
+"""
+    linear_map!(Zout::Zonotope, M::AbstractMatrix, Z::Zonotope)
+
+Compute the concrete linear map of a zonotope storing the result in `Zout`.
+
+### Input
+
+- `Zout` -- zonotope (output)
+- `M`    -- matrix
+- `Z`    -- zonotope
+
+### Output
+
+The zonotope `Zout`, which is modified in-place.
+"""
+function linear_map!(Zout::Zonotope, M::AbstractMatrix, Z::Zonotope)
```

Comment:
Does this also work if `Zout === Z`?

Reply:
no; in `mul!(Y, A, B)`, `Y` must not be aliased with either `A` or `B` (see `?mul!`)

Reply:
Makes sense.